internal/provider/resource_auth0_prompt.go

```diff
@@ -55,29 +53,25 @@ func createPrompt(ctx context.Context, d *schema.ResourceData, m interface{}) di
 
 func readPrompt(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	api := m.(*management.Management)
+
 	prompt, err := api.Prompt.Read()
 	if err != nil {
-		if mErr, ok := err.(management.Error); ok {
```

Comment:
Is this not necessary for this operation?

Reply:
This was actually unnecessary because if we are getting a 404 it means the endpoint disappeared. It's not an endpoint of an ID managed resource: https://auth0.com/docs/api/management/v2#!/Prompts/get_prompts.